docs/kbs_attestation_protocol.md

```diff
@@ -183,7 +183,8 @@ If KBS verified the token or the attestation evidence successfully, it will retu
 	    /* Symmetric key and algorithm information for encrypting `output`. */
 	    "crypto-annotation": {
             "algorithm": "",
-            "symkey-length": "",
+			/* The symmetric encryption IV, if the algorithm used does not need IV, this field left blank */
+            "iv": "",
```

Comment:
Hi @jialez0 - Please can you provide further detail in the commit for this change. For example:

- What is "iv"? (What does it stand for? Is it https://en.wikipedia.org/wiki/Initialization_vector?)
- Is "iv" an exact replacement for "symkey-length"? If the URL above is correct, I think the answer is no since one is a length, and the other appears to be a blob of random data, like a salt?
- iff "iv" == "Initialization vector", why can't we call the member "initialization-vector" (or "initialization-vector-blog or similar). That would be much clearer and consistent with all the other members that are human-readable.
- Is there a canonical reference URL to explain this change (official doc(s) or an GitHub issue?

Also, please can you add an entry for "IV" to https://github.com/confidential-containers/documentation/wiki/Acronyms.

Reply:
Sorry for the confusing, I have changed the name `iv` to `initialization-vector` and we don't need to use the abbreviation anymore, That does cause some confusion.